pypeit/reduce.py

```diff
@@ -252,12 +255,16 @@ def extract(self, global_sky, sobjs_obj):
             # Quick loop over the objects
             for iobj in range(self.sobjs.nobj):
                 sobj = self.sobjs[iobj]
-                plate_scale = self.get_platescale(sobj)
+                bin_spec, bin_spat = parse.parse_binning(self.binning)
+                plate_scale = self.get_platescale(sobj)*bin_spat
```

Comment:
Can we change the method here to get_pixelscale, and put this parse binning command in that method.